src/__tests__/App.test.tsx

```diff
@@ -8,13 +8,12 @@ import { render, screen, FetchMockSandbox } from './utils/test-utils';
 describe('App', () => {
   beforeEach(() => {
     const { testData } = getTestData();
-    (global.fetch as FetchMockSandbox)
-      .get('glob:https://treeherder.mozilla.org/api/project/*/push/*', {
-        results: [],
-      })
```

Comment:
When requesting a single hash, we should always get a single result. And our code relies on that (maybe we should change our code that that's another story).

Reply:
Hmm, that's something to keep note of in the future.